README.md

```diff
@@ -15,3 +15,42 @@ npm start
 ```
 
 The app will be live at http://localhost:19002.
+
+## Backend setup
+
+Required:
+- [NodeJS](https://nodejs.org/en/download/)
+- [MongoDB Atlas](https://www.mongodb.com/cloud/atlas/register)
+
+Recommended:
+- [MongoDB Compass](https://www.mongodb.com/products/compass) - To view and work directly with your database
+- [Postman](https://www.postman.com/) 
+    - Can use it to test your API endpoints manually. 
+    - [![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/18831621-c8abd2e1-57b9-47df-9e51-2e2c76d4a031?action=collection%2Ffork&collection-url=entityId%3D18831621-c8abd2e1-57b9-47df-9e51-2e2c76d4a031%26entityType%3Dcollection%26workspaceId%3Daf8d3bf7-ac26-4763-8491-bfd7d6f04ae9)
+    - Please reach out to Aman if you want edit access to the postman collection
+
+To get started use a terminal, cd into the backend directory, install packages
+```
+cd backend
+npm install
+```
+
+Next, you can start the server in one of the two ways
+1. Running live development server: `npm run dev` (Recommended)
+2. Start the server without live refresh: `npm start`
+
+Location at which server will be hosted will be displayed on terminal. By default, it will be live at http://localhost:3000.
+
+We are using a mongoDB database named "LAK" hosted on MongoDB Atlas. Please ping Assaf to get yourself added to the database so that you can start working on the backend. 
+
+Here is the backend repo structure
+
+```bash
```

Comment:
Please review this repo layout and let us know if you want it changed.

Reply:
Yes, this looks great! very well organized.

Reply:
Looks good! Very easy to follow :)